scripts/setup-testsuite-dataset

```diff
@@ -3,9 +3,8 @@
 # the integration testsuite requires that a dataset and particular columns already exist
 # this little script handles that bit of bootstrapping in an idempotent fashion
 
-HONEYCOMB_API="${HONEYCOMB_API:=https://api.honeycomb.io}"
-COLUMNS_API="${HONEYCOMB_API}/1/columns"
-DATASETS_API="${HONEYCOMB_API}/1/datasets"
+COLUMNS_API="${HONEYCOMB_API_ENDPOINT}/1/columns"
```

Comment:
I think doing the default message below means that `COLUMNS_API` and `DATASETS_API` will get set to `/1/columns` because the variable is missing at the time of them being set.

Maybe easier to just do it this way? If you want to bring the nicer "using the default" style message up to the top that's great too :)

```suggestion
HONEYCOMB_API_ENDPOINT="${HONEYCOMB_API_ENDPOINT:=https://api.honeycomb.io}"
COLUMNS_API="${HONEYCOMB_API_ENDPOINT}/1/columns"
```

Reply:
Ah whoops! I think I didn't realize the `:=` syntax is an if unset type of thing but just looked it up and putting it back like this seems like the simplest/best thing.